base/int.jl

```diff
@@ -337,8 +337,8 @@ end
     mod(x, T)
     %(x, T)
 
-For an integer type `T`, finds `z::T` such that `x` ≡ `z` (mod n), where n is the
-cardinality of `T` and `z` is restricted to the interval of values representable by `T`.
+For an integer type `T`, finds `y::T` such that `x` ≡ `y` (mod n), where n is the
+number of elements representable by `T`, and `y` is in `[typemin(T),typemax(T)]`.
```

Comment:
Perhaps "number of values representable by `T`,"? ("elements" seems the right choice when referring to a set, e.g. "number of elements in the set of values of `T`"; values seems more appropriate coupled to "representable by `T`".)

Perhaps ", and `y` is restricted to the values representable by `T`"? (`y` not merely being restricted to the interval `[typemin(T), typemax(T)]`.)

Perhaps collapsing those phrases is possible: "where `y` is restricted to the values representable by `T`, and `n` is the number of such values" or so? Best!